projects/wipp-frontend-lib/src/lib/csv-collection/csv-collection.service.ts

```diff
@@ -1,21 +1,27 @@
-import { Injectable } from '@angular/core';
+import { Injectable, Inject } from '@angular/core';
 import {HttpClient, HttpHeaders, HttpParams} from '@angular/common/http';
 import {Observable} from 'rxjs';
 import {map} from 'rxjs/operators';
 import {Job} from '../job/job';
 import {CsvCollection, PaginatedCsvCollections} from './csv-collection';
-import {environment} from '../../environments/environment';
 import {DataService} from '../data-service';
 import {Csv, PaginatedCsv} from './csv';
+import { API_ROOT_URL } from '../injection-token';
+import { WippFrontendLibConfigurationProvider } from '../wipp-frontend-lib-configuration';
 
 @Injectable({
   providedIn: 'root'
 })
 export class CsvCollectionService implements DataService<CsvCollection, PaginatedCsvCollections> {
 
-  private csvCollectionUrl = environment.apiRootUrl + '/csvCollections';
+  private csvCollectionUrl = this.apiRootUrl + '/csvCollections';
+  private plotsUiUrl = this.configurationProvider.config.plotsUiUrl;
```

Comment:
Can we pass both variables through the same provider?

Reply:
the apiRootUrl is coming from environments.ts whereas plotsUiUrl is imported trough the configuration provider  from the config,json file